week2/homework/index.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const fs = require('fs');
+
+let fileName = 'toDo.txt';
+// console.log(fileName);
+function readFile() {
+    return new Promise(
+        resolve => fs.readFile(
+            fileName,
+            (err, data) => resolve(err ? '' : data.toString())
+        )
+    );
+}
+
+function remove(...i) {
+    readFile()
+        .then(content => {
+             content.splice(i, i+1);
+        writeFile(content)    
```

Comment:
`writeFile` returns a promise that should return inside then.

Also try indent your code correctly.